mods/ts/sequences/structures.yaml

```diff
@@ -846,7 +846,7 @@ nahpad:
 		Offset: 0, 0
 		UseTilesetCode: false
 
-proc: # TODO: unused narefn_a, narefn_b
+proc:
```

Comment:
True, but at least the utility reminds people that there are TODOs that still need doing. Unless you want to permamently get rid of that animation?